index.html

```diff
@@ -53,6 +53,28 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
         </div>
       </div>
       <!-- end copy -->
+      <div class="col-md-4">
+        <div class="profile-section">
+          <div class="text-center">
+            <img src="https://c1.staticflickr.com/3/2758/4060282653_1fc33aeea9_b.jpg" alt="Shehroz Irfan" class="profile-image">
+            <hr>
+          </div>
+          <div class="profile-bio pt-2">
+            <div class="profile-name">Max Walter</div>
+            <div class="profile-skills">Software Engineer | iOS Dev</div>
+            <hr>
+            <div class="profile-github pt-2">
+              <p>
+                <a href="https://github.com/" target="_blank"><i
+                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
+                <a href="https://www.linkedin.com/in/" target="_blank"><i
+                    class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      
```

Comment:
Hey @Shaurya50211 !
Remove this extra line.

Reply:
Sure